backend/api/migrations/0015_rename_comment.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you adjust the filename for this so it's incremented by 1? Just to avoid two 15's